### PR TITLE
Fixes #20902: ensure 'katello' is webpack bundle name.

### PR DESCRIPTION
**Pull request by waldenraines**
We were removing the bundle name for katello and thus making it
difficult to reference the katello bundle.  This commit changes the
script to ensure that the katello bundle name is retained.

http://projects.theforeman.org/issues/20902

### CODE REVIEW

File: script/plugin_webpack_directories.rb
Comment (waldenraines) [inline script/plugin_webpack_directories.rb:33]: The real question here is should we be doing this even for foreman?  I would argue that the bundle name of foreman should probably be 'foreman' and not ''.

**Comment by ohadlevy** [comment]
thanks @waldenraines overall :+1: can you please add a test at https://github.com/theforeman/foreman/blob/develop/test/unit/script/plugin_webpack_directories_test.rb ?

File: script/plugin_webpack_directories.rb
Comment (waldenraines) [reply script/plugin_webpack_directories.rb:33]: @ohadlevy thoughts on this?

**Review by dLobatog — CHANGES_REQUESTED** [review]
@waldenraines Mind to rebase with latest develop? It would make the katello tests pass - as they need a change recently merged

File: script/plugin_webpack_directories.rb
Comment (waldenraines) [reply script/plugin_webpack_directories.rb:33]: In other words I would prefer to just remove this line.  I think it makes sense for foreman to reference its own bundle as 'foreman' just like everything else.

**Comment by waldenraines** [comment]
> @waldenraines Mind to rebase with latest develop? It would make the katello tests pass - as they need a change recently merged

Done.

**Review by dLobatog — APPROVED** [review]

**Comment by waldenraines** [comment]
This is going to conflict with https://github.com/theforeman/foreman/pull/4733 so we should decide which is going first.

File: script/plugin_webpack_directories.rb
Comment (ohadlevy) [reply script/plugin_webpack_directories.rb:33]: @waldenraines do you mind if we just merge as is to unblock katello? changing the bundle name might have other impact on webpack configuration etc which i think we can look into in another pr?

**Review by dLobatog — APPROVED** [review]
I'll be merging this to unblock Katello, the `bundle_name` thing can be resolved in other PR if needed.

**Pull request merged**
